README.md

```diff
@@ -1,26 +1,94 @@
-# nats-operator
+# NATS Operator
 
 [![Build Status](https://travis-ci.org/nats-io/nats-operator.svg?branch=master)](https://travis-ci.org/nats-io/nats-operator)
 
-`nats-operator` manages NATS clusters atop [Kubernetes][k8s-home], automating their creation and administration.
+NATS Operator manages NATS clusters atop [Kubernetes][k8s-home], automating their creation and administration.
 
-**Project status: *ALPHA***
-
-The API, spec, status and other user facing objects are subject to change.
-We do not support backward-compatibility for the alpha releases.
+[k8s-home]: http://kubernetes.io
 
 ## Requirements
 
-- Kubernetes 1.7+
+- Kubernetes v1.7+
```

Comment:
Actually, after your [change here](https://github.com/nats-io/nats-operator/pull/23/files#diff-e87d9ff03678a45cd0da2aca55f6fe9cR1) this will only work on Kubernetes 1.8 or newer.

Reply:
thanks, yes should bump in the readme then